shared/js/browser/windows-communication.js

```diff
@@ -119,6 +124,7 @@ function handleViewModelUpdate(viewModel) {
     certificateData = viewModel.certificates || [];
     protections = viewModel.protections;
     locale = viewModel.localeSettings?.locale;
+    maliciousSiteStatus = viewModel.maliciousSiteStatus;
```

Comment:
I think this should be `maliciousSiteStatus = viewModel.maliciousSiteStatus || { "kind": null };` to keep backwards compat with pre-Malicious-site Windows browser version (the Privacy Dashboard seems to crash if this isn't provided)

Reply:
Setting that parameter to null would result in the same state as when the site is considered safe, meaning that in your version of the Windows app, the dashboard wouldn’t wait for the final state of maliciousSiteStatus before rendering, which would cause it to default to safe at first, and then switch to the unsafe state on malicious sites.

Seeing as how this change is only going out with the versions that support it, I think it’s safe to keep it that way. We’ve done the same on other platforms without issue.

If you’re still unsure about keeping it in though, we can discuss alternatives that don’t involve defaulting to null.